apps/ensadmin/src/components/registrar-actions/resolve-and-display-registrar-actions-panel.tsx

```diff
@@ -0,0 +1,31 @@
+import { useActiveNamespace } from "@/hooks/active/use-active-namespace";
+
+import { DisplayRegistrarActionsPanel } from "./display-registrar-actions-panel";
+import { useFetchRegistrarActions } from "./use-fetch-registrar-actions";
+
+interface ResolveAndDisplayRegistrarActionsPanelProps {
+  maxItems: number;
```

Comment:
```suggestion
  itemsPerPage: number;
```

Goal: Align with terminology we're building up in https://github.com/namehash/ensnode/pull/1239